### PR TITLE
Beginner tutorial

### PR DESCRIPTION
**Pull request by VictorMeneghini**
Created a tutorial for beginners users

### CODE REVIEW

File: src/components/repository/BeginnerTutorial.vue [hunk 1]
```diff
@@ -0,0 +1,182 @@
...
+    &__footer {
+      width: 100%;
+      text-align: center;
+
+      &__steps {
+        display: inline-block;
+        margin-top: .5rem;
+
+        &__step {
+        height: 15px;
```
Comment (douglasppaz) [inline src/components/repository/BeginnerTutorial.vue]: why 15px?

File: src/components/repository/BeginnerTutorial.vue [hunk 2]
```diff
@@ -0,0 +1,182 @@
...
+
+          &--active {
+            background-color: $color-primary;
+            opacity: 1;
+            animation-name: floating;
+            animation-duration: 1s;
+            animation-iteration-count: infinite;
+            animation-timing-function: ease-in-out;
+            @keyframes floating {
+              from { transform: translate(0, 0px); }
```
Comment (douglasppaz) [inline src/components/repository/BeginnerTutorial.vue]: use 0%

File: src/components/repository/BeginnerTutorial.vue [hunk 3]
```diff
@@ -0,0 +1,182 @@
...
+              65%  { transform: translate(0, -5px); }
+              to   { transform: translate(0, 00px); }
```
Comment (douglasppaz) [inline src/components/repository/BeginnerTutorial.vue]: use 100%

Comment (douglasppaz) [reply src/components/repository/BeginnerTutorial.vue]: just 0, not 00px

File: src/components/repository/BeginnerTutorial.vue (hunk 1 above)
Comment (VictorMeneghini) [reply src/components/repository/BeginnerTutorial.vue]: I will change to rem

File: bh/src/components/BhStepper.vue [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+      opacity: 0.5;
+
+      &--active {
+        background-color: $color-primary;
+        opacity: 1;
+        animation-name: floating;
+        animation-duration: 1s;
+        animation-timing-function: ease-in-out;
+        animation-iteration-count: infinite;
+        @keyframes floating {
```
Comment (douglasppaz) [inline bh/src/components/BhStepper.vue]: break line

File: src/components/repository/BeginnerTutorial.vue [hunk 5]
```diff
@@ -0,0 +1,177 @@
...
+        height: 1rem;
+        width: 1rem;
+        margin: 1rem;
+        background-color: #bbbbbb;
+        border: none;
+        border-radius: 50%;
+        display: inline-block;
+        opacity: 0.5;
+
+          &--active {
```
Comment (douglasppaz) [inline src/components/repository/BeginnerTutorial.vue]: remove this, stepper has this property

File: src/components/shared/Layout.vue [hunk 6]
```diff
@@ -87,6 +97,8 @@
     <new-repository-modal
       :active="newRepositoryModalOpen"
       @requestClose="closeNewRepositoryModal()" />
+    <BeginnerTutorial
```
Comment (douglasppaz) [inline src/components/shared/Layout.vue]: Use beginner-tutorial

**Pull request merged**